sql/src/SqlEval.sk

```diff
@@ -591,7 +631,23 @@ class Evaluator{options: Options, user: ?UserFile} {
           sinkDirName,
           (context, _writer, key, valueIter) ~> {
             src = SKStore.Path(selectDir.dirName, key);
-            entries = Vector[(key, (src, src, valueIter.collect(Array)))];
+            rows: Array<SKStore.File> = valueIter
```

Comment:
Typing fails without this annotation, but I haven't been able to find a better way to write this.